anomaly_detector.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import numpy as np
+from matplotlib import pyplot as plt
+from sklearn.metrics import r2_score
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import GradientBoostingRegressor
+from sklearn.linear_model import LinearRegression
+from datetime import timedelta, datetime
+
+
+class AnomalyDetector(object):

```

Comment:
Мы не во втором питоне, от object не наследуемся.
```suggestion
class AnomalyDetector:
```